server.ts

```diff
@@ -0,0 +1,36 @@
+import * as express from 'express';
+import { TodoApp } from './src/todo';
+
+const todo = new TodoApp();
+
+const serv = express();
+serv.use(express.json());
+
+const port = 3000;
+
+serv.get('/', function(req, res, next) {
+    res.json(todo.listAllToDos());
```

Comment:
So this is great.  But of note this will make debugging harder because this makes it hard to get the value of `listAllToDos` with a debugger.  So keep this in mind.  Variables are effectively free.